packages/components/src/index.js

```diff
@@ -22,6 +22,7 @@ import CollapsiblePanel from './CollapsiblePanel';
 import ConfirmDialog from './ConfirmDialog';
 import Datalist from './Datalist';
 import { ModelViewer, RecordsViewer } from './DataViewer';
+import { importFromCDN } from './importFromCDN';
```

Comment:
What do you think about a babel macro instead? Defined in TUI Scripts?

Reply:
AFAIK macro are good when you have few lines of code here I am not sure of the qtt